resources/views/admin/orders.blade.php

```diff
@@ -65,6 +65,9 @@
                     <th scope="col" class="w-1/5 px-6 py-3 text-start text-xs font-medium text-black uppercase dark:text-white">
                         {{ __('manage-orders/orders.date') }}
                     </th>
+                    <th scope="col" class="w-1/5 px-6 py-3 text-start text-xs font-medium text-black uppercase dark:text-white">
+                        {{ __('manage-orders/orders.group_name') }}
+                    </th>
```

Comment:
Aangezien dit nu in het overzicht toegevoegd is, vind ik het wel passend als hier een zoek- en/of sorteerfilter op komt. Ik heb het idee dat Ivo hier een opmerking over gaat geven (wat ik ook snap)